.travis.yml

```diff
@@ -5,11 +5,9 @@ before_script:
   - go get github.com/mattn/goveralls
 
 script:
-  - make lint
-  - sudo -E env "PATH=$PATH" make test
-  - sudo -E env "PATH=$PATH" make test-coverage
-  - for pkg in `ls test/coverage/*.cover`; do goveralls -coverprofile=$pkg -service=travis-ci; done
-  - make build
+  - make test
```

Comment:
make lint

Reply:
in this case, make lint will not perform any formatting tests (gofmt) 
make test will trigger both lint and gofmt.

if make test is not desirable i can explicitly run here:
- make fmt
- make lint
- make test-coverage

this will prevent running the tests as both with/without coverage.
when i move the project to golangci-lint ill change this to make lint.